docs/azurefunction_maven.md

```diff
@@ -0,0 +1,25 @@
+# Configure Maven for VS Code
+
+Maven is a software project management and comprehension tool, which could manage a project's build, reporting and documentation from a central piece of information. The VS Code Azure Function Plugin use maven to generate Java Function project. Please ensure you have already download maven and set the PATH correctlly before you create Java Functions.
```

Comment:
Could you make the following changes?

1. VS Code Azure Functions Extension (versus Plugin)
1. download => downloaded
1 . correctlly => correctly

Reply:
Done~